Scoop/Supporting/Utilities/Extension+UIViewController.swift

```diff
@@ -17,3 +17,18 @@ extension UIViewController {
     }
     
 }
+
+extension UIView {
+    
+    func addDropShadow(){
+        layer.shadowColor = UIColor.black.cgColor
+        layer.shadowOpacity = 0.3
+        layer.shadowRadius = 5
+        layer.shadowOffset = .zero
+        
+        //MARK: Caches shadow, so doesn't need to be redrawn
+        layer.shouldRasterize = true
```

Comment:
actually to add on, i think rasterizing sometimes makes things appear more pixelated. does it look alright in your case?

Reply:
I don't think I noticed a difference when I was testing it, but I didn't see any issues in performance when I commented it out, so maybe I'll just remove/ comment it out for now.